lib/database_cleaner/active_record/truncation.rb

```diff
@@ -67,7 +67,7 @@ def initialize(connection)
         when "SQLite"
           extend AbstractMysqlAdapter
           extend SQLiteAdapter
-        when "PostgreSQL"
+        when "PostgreSQL", "PostGIS"
```

Comment:
I would say the simple change would be a great, small change to support PostGIS ActiveRecord adapter (which inherits from the PostgreSQL adapter).

The other two changes in this Pull Request are perhaps a bit more optional, and really come down to how willing the maintainers of this gem are to consider the PostGIS adapter officially supported (tested against, etc)?